client/src/telethon/_impl/client/types/keyboard.py

```diff
@@ -55,4 +55,24 @@ def __init__(
         self._raw = types.ReplyInlineMarkup(rows=_build_keyboard_rows(buttons))
 
 
-KeyboardType: TypeAlias = Keyboard | InlineKeyboard
+class HiddenKeyboard:
+    __slots__ = ("_raw",)
+
+    def __init__(self, selective: bool) -> None:
+        self._raw = types.ReplyKeyboardHide(selective=selective)
+
+
+class ForcedReplyKeyboard:
```

Comment:
English is not my native language, please check that the naming is grammatically correct

Reply:
Yeah honestly not super sure what to call these. It's a shame "buttons" doesn't really work because it's not only inline and "normal", but also all these weird cases…

This is likely fine for now. It can always be renamed before v2 is out.